app/views/users/index.html.erb

```diff
@@ -1,7 +1,12 @@
-<h1>list of all users</h1>
-
 <ul>
   <% @users.each do |user| %>
-    <li><%= link_to user.name, user_path(user) %></li>
+    <li>
+      <%= link_to user.name, user_path(user) %>
```

Comment:
- [ ] Great job working on the requested changes but  I think it still there a problem with linking to the user show page since when I press on a user it doesn't lead me to the user show page for example user/1 it just shows me this error so please add the correct link for the user show page here to make your app navigatable 👏🏻♻💪🏻✨🔥


![image](https://github.com/axcell-0/Blog-app/assets/58553711/4314f2ef-3ba1-40e4-8eb3-3c8cc04439be)

- It should show like this 

![image](https://github.com/axcell-0/Blog-app/assets/58553711/d034dd25-62af-4792-9eb3-d91a7653e0c6)

- Please check the video when I press on a user it does nothing


https://github.com/axcell-0/Blog-app/assets/58553711/4fb76c77-775f-43ad-a237-2adb406c5906

Reply:
@Reem-lab Thank you very much for the review, please i have make research on how to fix the problem can you help me out please i will be sincerely gratefull